### DISCUSSION TITLE
Best Internal Business Instant Messenger

### DISCUSSION THREAD

**Discussion by McJul** [NAIST-SE/AttractingContributors#372] [Polls]
What do you prefer and why?

**Comment by ellielo** [comment]
I use Telegram mostly

**Comment by janply** [comment]
We used Teams for a long time. Then switched to Slack. I like slack much better. It's convenient for work. We can create closed and open groups, chat and call to one or more recipients, receive and [send sms from slack](https://clerk.chat/slack) directly to a phone number. Nevertheless, it should be tried and tested for your personal needs.
